src/Import/Remote/ComposerApplication.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Castor\Import\Remote;
+
+use Composer\Command\SelfUpdateCommand;
+use Composer\Console\Application as BaseApplication;
+
+/** @internal */
+final class ComposerApplication extends BaseApplication
```

Comment:
```suggestion
/** @internal */
#[Exclude]
final class ComposerApplication extends BaseApplication
```

donc forget the `use`